ReactTask/src/modules/AlertMessage.models.ts

```diff
@@ -0,0 +1,5 @@
+export interface IAlertMessage {
+  textMessage: string;
+  statusErr: boolean | any;
```

Comment:
Idk if u can do this but could you write just statusErr?: boolean. Would it solved the problem?

Reply:
It was just the first file I changed and remembered how everything worked
Fixed